docs/source/DeprecationPage.md

```diff
@@ -0,0 +1,21 @@
+# Deprecation
+
+
+## Kokkos-4.0 and Kokkos-3.7.0
```

Comment:
It is not clear to me if these are the things marked for deprecation in 3.7 or 4.0 ... could we maybe have them in separate tables?

Reply:
These are features we deprecated in 3.7.00. Hence, I read that as "between Kokkos 3.7.0 and Kokkos 4.0"